pymoveit2/robots/crane_x7.py

```diff
@@ -0,0 +1,35 @@
+from typing import List
+
+MOVE_GROUP_ARM: str = "arm"
+MOVE_GROUP_GRIPPER: str = "gripper"
+
+OPEN_GRIPPER_JOINT_POSITIONS: List[float] = [0.7, 0.7]
+CLOSED_GRIPPER_JOINT_POSITIONS: List[float] = [0.0, 0.0]
+
+robot_prefix = "crane_x7_"
```

Comment:
```suggestion
robot_prefix = "crane_x7_"

```

Reply:
Thank you for your feedback @AndrejOrsula! I'll make sure to format from now on :)